mypy.ini

```diff
@@ -0,0 +1,17 @@
+[mypy]
+mypy_path = stubs
+#warn_return_any = True
```

Comment:
Remove these commented out things

Reply:
in config files, commented lines provide context about what other options can be enabled.  I will not remove those lines.

I might uncomment them and change the boolean value to False.  But probably not, as that's pretty pointless.

Reply:
That's a stupid reason. That's what documentation is for. I don't need you to comment out every single option that MyPy can take.

Reply:
No, I'm going to leave the ones I might want to change at some point.

Reply:
Fine, but that's stupid. I'll let you off because this PR is quite epic.